### PR TITLE
Init repo

### PR DESCRIPTION
**Pull request by QuentinBisson**

### CODE REVIEW

**Comment by mig4** [comment]
Is this generated? If so how?

**Comment by QuentinBisson** [comment]
I copied the init from prometheus rules actually. I have a PR in  giantswarm/github as well

**Comment by QuentinBisson** [comment]
https://github.com/giantswarm/github/pull/224

**Comment by mig4** [comment]
Yip, seen it, approved.
But doesn't this mean these changes should/will be generated by automation and a PR proposed by architect bot?

**Comment by QuentinBisson** [comment]
Some will be yes and they will be overriden but it can take some time for it to run

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  - package-ecosystem: github-actions
+    directory: "/"
+    schedule:
+      interval: weekly
+      time: "04:00"
+    open-pull-requests-limit: 10
+    reviewers:
+      - giantswarm/team-atlas
+    ignore:
+      - dependency-name: zricethezav/gitleaks-action
```
Comment (mig4) [inline .github/dependabot.yml:16]: Just checking, why ignore it?

Comment (QuentinBisson) [reply .github/dependabot.yml:16]: This is from a generated file so I'm not really sure :(

**Review by TheoBrigitte — APPROVED** [review]
No need for approval for generated files

**Pull request merged**